src/services/tx/relaying.ts

```diff
@@ -19,7 +19,6 @@ type SponsoredCallResponse = {
 export type RelayResponse = {
   limit: number
   remaining: number
-  expiresAt?: number
```

Comment:
I assume this was removed on the relay service and we are updating it here?

Reply:
Yeah, saw it and thought to update it.